src/test/expected/one-assert/test.rs

```diff
@@ -1,7 +1,9 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0 OR MIT
-
-pub fn main() {
+// rmc-flags: --function check_assert
+// compile-flags: --crate-type lib
```

Comment:
Why is this needed?

Reply:
This is needed because binary crates have more checks than library crates. The purpose of this test is to ensure that there is only one assertion in the result.

Reply:
I'm curious what the extra checks for binary crates are.